### PR TITLE
Task 41902 : Add FIDO Integration for MFA

### PR DESCRIPTION
**Pull request by rdenarie**

### CODE REVIEW

File: webapp/portlet/src/main/webapp/mfa-access/components/FidoAccess.vue [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+                }).then(attestation => {
+                  this.finishRegistration(attestation);
+                }).catch((err) => {
+                  this.changeScreen('error');
+                  console.error('Unable to create credentials', err);
+                });
+              }
+            });
+          } else {
+            this.changeScreen('success');
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/mfa-access/components/FidoAccess.vue]: why do you change here ?
It is an error case

File: webapp/portlet/src/main/webapp/mfa-access/components/FidoAccess.vue [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+        credentials: 'include',
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded',
+          'pragma': 'no-cache',
+          'cache-control': 'no-cache',
+        },
+        body: formData
+      }).then(resp => {
+        if (resp && resp.ok) {
+          console.log('Registration success');
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/mfa-access/components/FidoAccess.vue]: Here we have a success case, should we display success screen ?

Comment (ahmadmido) [reply webapp/portlet/src/main/webapp/mfa-access/components/FidoAccess.vue]: Done !

**Review by ahmadmido — DISMISSED** [review]
LGTM

**Review by ahmadmido — APPROVED** [review]

**Pull request merged**